builds/serverless-config-pack/serverless-go.yml

```diff
@@ -102,6 +102,12 @@ functions:
 #Start:resources
 resourcesDisabled:
   Resources:
+    #Start:isVaultEnabled
+    customEventRole: ${file(policyFile.yml):vaultRole}
```

Comment:
`customEventRole` is definitely misleading.. :)

Reply:
Shouldn't this just be `basicRole` &  `vpcRole` instead?

Reply:
Changed